lib/screens/settings/tts.dart

```diff
@@ -262,7 +262,14 @@ class TextToSpeechScreen extends StatelessWidget {
               onChanged: (value) {
                 model.isPreludeMuted = value;
               },
-            )
+            ),
+            SwitchListTile.adaptive(
```

Comment:
I don't think we should offer this as an option, nobody will switch to it. We should improve tts based on the feedback and then that would be enough.

Reply:
In particular it seems like most people complained about the speed control not working and tts reading bot messages.